srcpkgs/pcem/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+pcem stores BIOS ROM images (not included),
+configuration files, and other data in ~/.pcem.
```

Comment:
This is documentation for the package and I don't believe it should be included in an INSTALL.msg.